charts/devtron/README.md

```diff
@@ -24,32 +24,23 @@ It packages third party components like
 
 ### Install with Helm
 
-This chart is currently not available on the official helm repository therefore you need to download it to install it.
+Run the following command to install the latest version of Devtron along with the CI/CD module:
```

Comment:
Run the following command to install the latest version of Devtron along with CI/CD integration:

Reply:
done